src/com/lftechnology/java/training/sanish/application/component/EmployeeHelper.java

```diff
@@ -0,0 +1,416 @@
+            int updateRow = 0;
+            String role;
+            String SetStatement;
+            String whereCondition = "employeeId=?";
+            EmployeeDao employeeDao = new EmployeeDao();
+            while (updateRow == 0) {
+                message = "\n >>" + Constants.TYPE_ROLE_LABEL;
+                LOGGER.log(Level.INFO, message);
+                role = UserInput.getString(inputScanner);
+                if (role.equals(Constants.ADMIN_ROLE) || role.equals(Constants.USER_ROLE)) {
+                    DateFormat dateFormat = new SimpleDateFormat(Constants.DB_DATE_FORMAT);
+                    Date date = new Date();
+                    SetStatement = "role=?, modifiedAt=?";
+                    updateRow = employeeDao.update(whereCondition, SetStatement, role, dateFormat.format(date), employee.getEmployeeId());
+                    if (updateRow == 0) {
+                        message = "\n" + Constants.ERROR_MSG_LABEL + Constants.FAIL_UPDATE_ROLE_MSG + Constants.TRY_AGAIN_MSG;
+                        LOGGER.log(Level.INFO, message);
+                        edit = UserInput.getString(inputScanner);
+                        if (!edit.equals(Constants.YES_OPTION) && !edit.equals(Constants.FULL_YES_OPTION)) {
+                            break;
+                        }
+                    }
+                } else {
+                    message = "\n" + Constants.ERROR_MSG_LABEL + Constants.INVALID_ROLE_MSG + Constants.TRY_AGAIN_MSG;
+                    LOGGER.log(Level.INFO, message);
+                    edit = UserInput.getString(inputScanner);
+                    if (!edit.equals(Constants.YES_OPTION) && !edit.equals(Constants.FULL_YES_OPTION)) {
+                        break;
+                    }
+                }
+
+            }
+        }
+    }
+
+    /**
+     * Change password of user
+     *
+     * @param inputScanner {@link Scanner}
+     * @param user         {@link User}
+     * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+     */
+    public static void changePassword(Scanner inputScanner, User user) {
+        Boolean changePassword = true;
+        UserDao userDao = new UserDao();
+        while (changePassword) {
+            String message = "\n >>" + Constants.TYPE_OLD_PASSWORD_LABEL;
+            LOGGER.log(Level.INFO, message);
+            String oldPassword = UserInput.getString(inputScanner);
+            message = "\n >>" + Constants.TYPE_NEW_PASSWORD_LABEL;
+            LOGGER.log(Level.INFO, message);
+            String newPassword = UserInput.getString(inputScanner);
+            message = "\n >>" + Constants.TYPE_RE_PASSWORD_LABEL;
+            LOGGER.log(Level.INFO, message);
+            String confirmPassword = UserInput.getString(inputScanner);
+            if (newPassword.equals(confirmPassword)) {
+                if (user.getPassword().equals(oldPassword)) {
+                    if (oldPassword.equals(user.getPassword())) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS134)